### PR TITLE
feat(capitalize): add capitalize

### PR DESCRIPTION
**Pull request by babay123**
I have implemented a `capitalize` function

### CODE REVIEW

File: src/string/capitalize.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ *
+ * @param {string} str - The string to be converted to uppercase.
+ * @returns {string} - The capitalized string.
+ *
+ * @example
+ * const result = capitalize('fred') // returns 'Fred'
+ * const result2 = capitalize('FRED') // returns 'Fred'
+ */
+
+export const capitalize = (str: string): string => {
```
Comment (Scalahansolo) [inline src/string/capitalize.ts:12]: The return type on this could be accurately be set to `Capitalize<string>`.

Comment (babay123) [reply src/string/capitalize.ts]: Thank you for code review, updated type, can you please review again

Comment (Scalahansolo) [reply src/string/capitalize.ts]: I'm just some random dude who is excited about this project and was poking around at the PRs, I have no authority to approve this code. I just thought it was simple improvement that could be made. (:

File: docs/ko/reference/string/capitalize.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# capitalize
+
+문자열의 첫 번째 문자를 대문자로 변환하고 나머지 문자는 소문자로 변환합니다.
```
Comment (raon0211) [inline docs/ko/reference/string/capitalize.md:3]:
```suggestion
문자열의 첫 번째 문자를 대문자로 변환하고, 나머지 문자는 소문자로 변환해요.
```

**Review by raon0211 — APPROVED** [review]
Thanks!

**Pull request merged**